crates/hc_sandbox/src/bundles.rs

```diff
@@ -0,0 +1,123 @@
+//! Helpers for working with dna files.
+use std::path::Path;
+use std::path::PathBuf;
+
+use anyhow::bail;
+use anyhow::ensure;
+use walkdir::WalkDir;
+
+#[derive(Debug, Clone)]
+/// Either a set of Dnas or a hApp bundle.
+pub enum DnasHapp {
+    /// Set of Dnas to install.
+    Dnas(Vec<PathBuf>),
+    /// Happ bundle to install.
+    HApp(Option<PathBuf>),
+}
```

Comment:
This type isn't used anywhere, is there a future plan for it, or can this file be removed?